anime_downloader/commands/dl.py

```diff
@@ -0,0 +1,126 @@
+import click
+import logging
+import os
+
+from anime_downloader import util
+from anime_downloader import session
+from anime_downloader.sites import get_anime_class
+from anime_downloader.__version__ import __version__
+
+logger = logging.Logger(__name__)
+
+echo = click.echo
+
+
+# NOTE: Don't put defaults here. Add them to the dict in config
+@click.command()
+@click.argument('anime_url')
+@click.option(
+    '--episodes', '-e', 'episode_range', metavar='<int>:<int>',
+    help="Range of anime you want to download in the form <start>:<end>")
+@click.option(
+    '--url', '-u', type=bool, is_flag=True,
+    help="If flag is set, prints the stream url instead of downloading")
+@click.option(
+    '--play', 'player', metavar='PLAYER',
+    help="Streams in the specified player")
+@click.option(
+    '--skip-download', is_flag=True,
+    help="Retrieve without downloading")
+@click.option(
+    '--download-dir', metavar='PATH',
+    help="Specifiy the directory to download to")
+@click.option(
+    '--quality', '-q', type=click.Choice(['360p', '480p', '720p', '1080p']),
+    help='Specify the quality of episode. Default-720p')
+@click.option(
+    '--fallback-qualities', '-fq', cls=util.ClickListOption,
+    help='Specifiy the order of fallback qualities as a list.')
+@click.option(
+    '--force-download', '-f', is_flag=True,
+    help='Force downloads even if file exists')
+@click.option(
+    '--file-format', '-ff', default='{anime_title}/{anime_title}_{ep_no}',
+    help='Format for how the files to be downloaded be named.',
+    metavar='FORMAT STRING'
+)
+@click.option(
+    '--provider',
+    help='The anime provider (website) for search.',
+    type=click.Choice(['9anime', 'kissanime', 'twist.moe', 'animepahe', 'kisscartoon', 'masterani', 'gogoanime'])
```

Comment:
@vn-ki  this is the part I meant, it would be easier if it was just put into its own variable. Simple change but convenient.